File: api/routers/misc.py
import time
from fastapi import APIRouter

router = APIRouter()


@router.get("/")
def health_check():
    return {
        "status": "healthy",
        "message": "Nightlio API is running",
        "timestamp": time.time(),
    }


@router.get("/time")
def get_current_time():
    return {"time": time.time()}

File: functions/_shared/routes/misc.ts
import { Hono } from 'hono';
import type { AppEnv } from '../lib/env';

export const miscRoutes = new Hono<AppEnv>();

miscRoutes.get('/', (c) =>
  c.json({
    status: 'healthy',
    message: 'Waymark API is running',
    timestamp: Date.now() / 1000,
  })
);

miscRoutes.get('/time', (c) => c.json({ time: Date.now() / 1000 }));
